FindBISON.cmake

```diff
@@ -76,146 +76,147 @@
 #=============================================================================
 
 find_program(BISON_EXECUTABLE NAMES bison win-bison win_bison DOC "path to the bison executable")
-MARK_AS_ADVANCED(BISON_EXECUTABLE)
```

Comment:
For vendored .cmake files it might make sense to skip them? But no super strong opinion, so feel free to ignore as it might be more hassle than worth.

Comparing local cmake-3.18's FindBISON.cmake, they minimally did lower-case, so our version is probably out-dated?

Reply:
I removed that and FindPackageMessage.cmake as well. I opened a ticket a few days ago about removing the vendored files since they should be available in every version of CMake we use now, so I'll probably open a PR for that next.